presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.tests.hive;
+
+import com.teradata.tempto.ProductTest;
+import com.teradata.tempto.query.QueryExecutor;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.TestGroups.HIVE_CONNECTOR;
+import static com.facebook.presto.tests.utils.QueryExecutors.connectToPresto;
+import static com.teradata.tempto.assertions.QueryAssert.assertThat;
+import static java.lang.String.format;
+
+public class TestGrantRevoke
+    extends ProductTest
+{
+    /*
```

Comment:
My current code tests for `SELECT, INSERT` and `DELETE` operations, which respectively requires `SELECT, INSERT` and `DELETE` privileges. Presto doesn't support `UPDATE` statement and hence, `UPDATE` privilege, so the code doesn't test it. We could add a negative test for it. But the test doesn't have to do with the `UPDATE` privilege per say. It would rather be a test for